### PR TITLE
Setup project

### PR DESCRIPTION
**Pull request by evans22j**
In this project, the following were implemented;
- Create a new directory and initialize it with git.
- Use the npm init -y command to create a package.json file.
- Create the entry point for index.js and index.css
- Create the main index.html file. Link your index.js and index.css files there.
- Create a directory called modules.
- Use ES6 syntax.
- Divide the code into modules. Save each module in a separate file in the modules dir. 
- Keep all functionalities of the app without errors.
- Refactor your methods and functions to arrow functions.
- Use let and const in the correct way.
- Display the current date below the navigation bar.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    });
+    localStorage.setItem('books', JSON.stringify(this.books));
+  }
+}
+
+// create an object
+const listBooks = new Books();
+function onPageReload() {
+  displayBook.innerHTML = listBooks.books.map((book, index) => `
+  <div class="display">
```
Comment (thecodechaser) [inline app.js]: - [ ] Please change this function to the arrow function to follow ES6 syntax.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+onPageReload();
+
+document.querySelector('span').innerHTML = new Date();
+
+const section = document.querySelectorAll('.section');
+const addNew = document.querySelector('.list-books');
+
+function makeActive(className) {
+  section.forEach((item) => {
+    if (item.classList.contains(className)) {
```
Comment (thecodechaser) [inline app.js]: - [ ] Please change this function to the arrow function to follow ES6 syntax.

File: index.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+const displayBook = document.querySelector('.display-books');
+const form = document.querySelector('form');
+
+// create an object
+const listBooks = new Books();
+function onPageReload() {
+  displayBook.innerHTML = listBooks.books
+    .map((book) => (
+  <div class='display'>
```
Comment (thecodechaser) [inline index.js]: - [ ] Please change this function to the arrow function to follow ES6 syntax.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,121 @@
+body {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
```
Comment (thecodechaser) [inline styles.css]: - [ ] For this project you are supposed to name the styles file called `index.css`. Please change your styles file name to `index.css` from `style.css`. 

### See the image for reference

image

File: modules/app.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+const section = document.querySelectorAll('.section');
+const addNew = document.querySelector('.list-books');
+
+function makeActive(className) {
+  section.forEach((item) => {
+    if (item.classList.contains(className)) {
+      item.classList.add('show');
+      addNew.classList.toggle('show');
```
Comment (thecodechaser) [inline modules/app.js]: - [ ] Please change this function to the arrow function to follow ES6 syntax.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @evans22j👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 4 above)
Comment (evans22j) [reply styles.css]: Done

File: modules/app.js (hunk 5 above)
Comment (evans22j) [reply modules/app.js]: done

File: index.js (hunk 3 above)
Comment (evans22j) [reply index.js]: done

File: app.js (hunk 2 above)
Comment (evans22j) [reply app.js]: done

File: app.js (hunk 1 above)
Comment (evans22j) [reply app.js]: done

Comment (Lordkaito) [reply app.js]: - [ ] As the previous reviewer mentioned, please change this function to make it an arrow function instead. Basically, what you need to do is get rid of the `function` keyword and do something like `const myFunction = () => { your code }`

File: app.js (hunk 2 above)
Comment (Lordkaito) [reply app.js]: - [ ] Also here, please apply the same changes to this function to make it an arrow function instead.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @evans22j, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omar-labana — COMMENTED** [review]
# **Approved ✔️**

Hi @evans22j,

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**